indigo_api/templatetags/indigo.py

```diff
@@ -91,3 +91,75 @@ def publication_document_description(work, placeholder=False):
         }
     elif placeholder:
         return _('Published')
+
+
+@register.simple_tag
+def work_commencement_description(work, friendly_date=True):
+    """ Describes the commencement status of a work.
+        Returns a tuple containing: (code, descriptive string, [optional commencing work tuple])
```

Comment:
I think it'll make reading and writing this code (and the code that uses the result) much cleaner if you return a dict instead:

```python
{
  "type": "uncommenced" # or "single" or "multiple" -- avoid magic numbers wherever possible :)
  "title": ...
  "note": ...
}

Reply:
Would it be useful to have this on the work object itself, rather than as a template tag?

Reply:
That would mean you can't pass in the `friendly_date` flag